setup.py

```diff
@@ -16,8 +16,8 @@
     keywords='Firebase',
     packages=find_packages(exclude=['tests']),
     install_requires=[
-        'requests', 'sseclient', 'gcloud',
-        'oauth2client', 'requests_toolbelt', 'python_jwt',
-        'pycrypto'
+        'requests==2.11.1', 'sseclient', 'gcloud==0.17.0',
+        'oauth2client==3.0.0', 'requests_toolbelt==0.7.0', 'python_jwt>=2.0.1',
```

Comment:
not really. just to avoid general backward compatibility issues.
we can be more flexible if needed.